libraries/lib-xml/XMLWriter.cpp

```diff
@@ -30,7 +30,10 @@ the general functionality for creating XML in UTF8 encoding.
 #include <wx/ffile.h>
 #include <wx/intl.h>
 
-#include <string.h>
+#include <cstring>
+#include <charconv>
```

Comment:
As you now know, <charconv> is incomplete for macOS and not even known in the Linux build.  Sorry, you must find an alternative to to_chars.

Reply:
I'm painfully aware of it now. This is really sad that a very simple part of C++17 is not implemented fully in 2021